Poker.BE/Poker.BE.Domain/Security/UserManager.cs

```diff
@@ -4,72 +4,89 @@
 using System.Text;
 using System.Threading.Tasks;
 using Poker.BE.Domain.Core;
+using Poker.BE.Domain.Utility.Exceptions;
 
 namespace Poker.BE.Domain.Security
 {
-    public class UserManager
-    {
-        #region Properties
-        protected Dictionary <string, User> UsersDictionary;
+	public class UserManager
+	{
+		#region Properties
+		protected Dictionary<string, User> UsersDictionary;
```

Comment:
👎 use `IDictionary<>` and not `Dictionary` as the type of the field / property. you need to use interfaces as the type for them. use the implementing class at assignment to `new`.